jib-core/src/main/java/com/google/cloud/tools/jib/configuration/FilePermissions.java

```diff
@@ -86,14 +86,23 @@ public static FilePermissions fromPosixFilePermissions(
   }
 
   /**
-   * Gets the corresponding permissions bits specified by the {@link FilePermissions}
+   * Gets the corresponding permissions bits specified by the {@link FilePermissions}.
    *
    * @return the permission bits
    */
   public int getPermissionBits() {
     return permissionBits;
   }
 
+  /**
+   * Gets the octal string representation of the permissions.
+   *
+   * @return the octal string representation of the permissions
+   */
+  public String toOctalString() {
```

Comment:
Maybe this should just be `toString`?

Reply:
I think `toOctalString` is better. There can be other string representation, like binary bits or just a human readable "rwxr--r--".